pkg/pillar/cmd/zedrouter/flowstats.go

```diff
@@ -590,6 +596,18 @@ func flowPublish(ctx *zedrouterContext, flowdata *types.IPFlow, seq, idx *int) {
 	*idx = 0
 }
 
+func checkFlowUnpublish(ctx *zedrouterContext) {
+	now := time.Now().Unix()
+	for k, m := range ctx.flowPublishMap {
+		passed := now - m
```

Comment:
I don't know what happens to Unix() is the clock is adjusted. Using time.Now() and time.Since() is robust against changes to the clock based on the golang documentation.

Reply:
Sure.